app/helpers/item_seasons_helper.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module ItemSeasonsHelper
```

Comment:
@vasconsaurus Before merging, please do a final check to delete files that are empty/we're not using :)